templates/views/beerProfile.html

```diff
@@ -0,0 +1,102 @@
+
+{% extends "templates/layouts/default.html" %}
+
+
+{% block intro %}
+
+{% endblock %}
+
+{% block content %}
+
+</script>
+<div class="bodyToLoad">
+	{% for beer in data.allBeers %}
+			<div class="row">
+				
+				<div class="col-md-4">	
+					<h4>Beers -> {{beer.beerName}}</h4>		
+				</div>
+
+				<div class="col-md-2 col-md-offset-6" style="border:1px solid black">
+					<span>order this beer</span>
+				</div>
+
+			</div>
+
+			<div class="row">
+				<div class="col-md-12" align="center">
+					<h1>{{beer.beerName}}</h1>
+				</div>
+			</div>
+
+			<div class="row">
+				<div class="col-md-12" align="center">
+					<h4>{{beer.city}} {{beer.state}}</h4>
+				</div>					
+			</div>
+
+		<br />
+		<br />
+		<br />
+	
+		<div class="col-md-12">
+			<div class="row">	
+				<div class="col-md-6">
+					<h4>{{beer.profileHeadline}}</h4>
+					<p>{{beer.profileBody}}</p>
+				</div>
+
+
+				<div class="col-md-6">
+					{{beer.profileVideo}}
+				</div>
+			</div>
+		</div>
+
+		<br />
+		<br />
+		<br />
+
+
+		<div class="row">
+			<div class="col-md-6">
+				<div class="col-md-12">
+					<h4>Connect with the Brewer</h4>
```

Comment:
This should pull in the brewer data as they're associated, but we can put that in another pull request.